packages/icon-builder/test/index.js

```diff
@@ -120,14 +118,14 @@ describe('--svg-dir, --innerPath, --fileSuffix', function() {
   });
 
   after(function() {
-    temp.cleanupSync();
+    // temp.cleanupSync();
```

Comment:
It shouldn't be commented out - I was trying to view any artifacts from a failing test. I'll fix it.